src/renderer/components/mulmo_viewer.vue

```
@@ -124,13 +132,17 @@ const props = defineProps<Props>();
 const projectId = computed(() => props.project?.metadata?.id || "");
 const videoUrl = ref("");
 const audioUrl = ref("");
+const pdfData = ref();
 
 const downloadMp4 = async () => {
   return downloadFile("movie", "video/mp4", projectId.value + "_video.mp4");
 };
 const downloadMp3 = async () => {
   return downloadFile("audio", "audio/mp3", projectId.value + "_audio.mp3");
 };
+const downloadPdf = async () => {
```

ESLint

[error] 'downloadPdf' is assigned a value but never used. Allowed unused vars must match /^__/u.

(@typescript-eslint/no-unused-vars)